sdk/attestation/azure-security-attestation/README.md

```diff
@@ -33,8 +33,8 @@ pip install --pre azure-security-attestation
 
 ### Authenticate the client
```

Comment:
Not part of the diff, but I think line 13 might make more sense as:
```
For a more complete view of Azure libraries, see the [azure sdk python release page](https://aka.ms/azsdk/python/all).
```

Reply:
Makes sense. And since you've not seen this code at all, I consider anything in the SDK should be in-scope, not just this commit.

Reply:
I just realized as well that the capitalization should probably be "Azure SDK for Python", even though the casual lowercasing is pretty Pythonic

Reply:
:) <VBG />